python/lsstdesc/pipe/task/coadd_in_cells.py

```diff
@@ -60,12 +56,15 @@ class CoaddInCellsConfig(pipeBase.PipelineTaskConfig,
     """
     seed = Field(
         dtype=int,
-        # default=0,
+        default=0,
```

Comment:
I removed this from being commented out. Was there any particular reason why the default value had been commented out? Any value you pass via a config parameter will override this value at runtime, but a sensible default (if not 0 is still preferable). Is it required that different runs necessarily must have different seeds, and the presence of a default value can lead to running the task without providing a random seed?

Reply:
We need this to be a required argument.

For the metacal task we ran on DC2 the operators refused to set unique seeds, they left it at the default. I tried to convince them but I could not, they claimed that it was too difficult.  Making this a requirement is the only way I can think of to force them to generate seeds